source/compatibility.tex

```diff
@@ -2295,6 +2295,13 @@
 
 Subclause \ref{cwchar.syn} describes the changes.
 
+\pnum
+Headers \tcode{<cstddef>}~(\ref{cstddef.syn})\indexlibrary{\idxhdr{cstddef}}
```

Comment:
I would suggest only mentioning `<cstddef>` here; the other clauses herein don't mention the corresponding `<stddef.h>`. If we want to draw attention to this, perhaps D.4 would be the place to do it.